src/commands/record.rs

```diff
@@ -251,7 +270,7 @@ mod tests {
         let mut writer = LineWriter::new(Box::new(c));
 
         // Serialize and write the event.
-        write_asciicast_event(&mut writer, event_type, duration, data.as_bytes()).unwrap();
+        write_asciicast_event(&mut writer, event_type, duration, data.as_bytes(), false).unwrap();
```

Comment:
Let's branch here between `write_asciicast_event` and the new `write_raw_event`.

Reply:
How about `write_raw_output`?

Reply:
Sure, that works.